lib/merkDb/MerkDbInMemoryDecorator.js

```diff
@@ -114,6 +114,8 @@ class MerkDbInMemoryDecorator {
     // commit
     batch.commitSync();
 
+    this.db.flushSync();
```

Comment:
Maybe it's better to flush on process exit? Flushing every block could lead to high IO